alluxio/posix/config.py

```diff
@@ -0,0 +1,77 @@
+import yaml
+import os
+
+from alluxio.posix.const import Constants
+from alluxio.posix.exception import ConfigMissingError, ConfigInvalidError
+
+
+class ConfigManager:
+    def __init__(self, config_file_path='config/ufs_config.yaml'):
+        self.config_file_path = config_file_path
+        self.config_data = self._load_config()
+        self.validation_functions = {
+            Constants.OSS_FILESYSTEM_TYPE: self._validate_oss_config,
+            Constants.ALLUXIO_FILESYSTEM_TYPE: self._validate_alluxio_config,
+            Constants.S3_FILESYSTEM_TYPE: self._validate_s3_config
+        }
+
+    def _load_config(self):
```

Comment:
It is necessary to add a hot update interface. For example, if the user does not have a configuration file and wants to configure config through the API